ddtrace/internal/ipv6.py

```diff
@@ -0,0 +1,18 @@
+import socket
+from typing import TypeVar
+from typing import Union
+
+
+T = TypeVar("T")
+
+
+# Returns true if address is an IPv6 address
+def is_ipv6_hostname(hostname):
```

Comment:
can we keep this in `ddtrace/internal/agent.py`? I don't think it needs a whole module on its own.